05/style.css/global.css

```diff
@@ -0,0 +1,63 @@
+html {
+  font-size: 1em;
+}
+.body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+.container {
+  width: 90%;
+  margin: 0 auto;
+}
+header {
+  font-size: 1em;
+  padding: 1.1em;
+  min-height: 10vh;
+  width: 100%;
+}
+header ul {
```

Comment:
Lepiej jest stylować po klasach (najlepiej zgodnie z BEM) niż po znacznikach.
Przy dużych projektach jest to spore utrudnienie, bo tych samych znaczników występuje dużo :)